clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java

```diff
@@ -638,24 +638,25 @@ private static void executeDataRecovery(CommandLine cmd) {
     boolean isDebuggingEnabled = cmd.hasOption(Arg.DEBUG.toString());
     boolean isNonInteractive = cmd.hasOption(Arg.NON_INTERACTIVE.toString());
 
-    StoreRepushCommand.Params cmdParams = new StoreRepushCommand.Params();
-    cmdParams.setCommand(recoveryCommand);
-    cmdParams.setDestFabric(destFabric);
-    cmdParams.setSourceFabric(sourceFabric);
-    cmdParams.setTimestamp(timestamp);
-    cmdParams.setSSLFactory(sslFactory);
-    cmdParams.setUrl(url);
+    StoreRepushCommand.Params.Builder builder = new StoreRepushCommand.Params.Builder();
+    builder.setCommand(recoveryCommand);
```

Comment:
@lyssarzr, thank you for fixing this issue! Maybe the more common for builder pattern is to have a chain of methods that each can return a reference to `this`,so that when you use it, you can write something like the following:

```
StoreRepushCommand.Params.Builder builder = new StoreRepushCommand.Params()
.setCommand(recoveryCommand)
.setDestFabric(destfabric)
.setSourceFabric(sourceFabric)
.....;
```

Reply:
@lluwm thanks for the review! does this mean we should re-expose the setter methods for params, undo the Builder class? gaojie mentioned above that maybe we should not expose those.

https://www.digitalocean.com/community/tutorials/builder-design-pattern-in-java this has been my reference for builder pattern also

Reply:
Oh, sure, we can follow the pattern and keep the new Builder class, but maybe we want something like this:

```
StoreRepushCommand.Params.Builder params = new StoreRepushCommand.Params.Builder()
.setCommand(recoveryCommand)
.setDestFabric(destfabric)
.setSourceFabric(sourceFabric)
.....;
```
for that, we may need to add `return this;` and let it return a `Builder`
to StoreRepushCommand.Builder.setCommand

```
public Builder setCommand(String command) {
        this.command = command;
        retur this;
 }
```